### PR TITLE
LDAS build

### PR DESCRIPTION
**Pull request by jread-usgs**
@aappling-usgs can you review and merge?

### CODE REVIEW

**Comment by aappling-usgs** [comment]
your commits include mine. does it matter whether or how we merge these two open PRs?

File: 1_ldas_host/in/ldas_subset.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  sub_variables: ['pressfc','dswrfsfc']
+  ldas_url: "dods://hydro1.sci.gsfc.nasa.gov/dods/NLDAS_FORA0125_H.002"
+  sub_split: 1000
+GLDAS:
+  sub_variables: ['psurf','swdown']
+  ldas_url: "dods://hydro1.sci.gsfc.nasa.gov:80/dods/GLDAS_NOAH025SUBP_3H"
+  sub_split: 1000
+  units: 
+    swdown: W m^-2
+    psurf: Pa
```
Comment (aappling-usgs) [inline 1_ldas_host/in/ldas_subset.yml]: i like the new yml format

**Comment by jread-usgs** [comment]
You OK if I merge yours?
I should have left it alone, but wanted to check for conflicts in the build

**Comment by aappling-usgs** [comment]
sure, go for it.

**Pull request merged**